packages/react-sdk/src/components/Whiteboard/SvgCanvas/SvgCanvas.tsx

```diff
@@ -108,6 +109,71 @@ export const SvgCanvas = forwardRef(function SvgCanvas(
     setContainerDimensions({ width, height });
   }, [width, height, setContainerDimensions]);
 
+  const handleCanvasClick = useCallback(() => {
+    if (svgRef.current) {
+      svgRef.current.focus();
+    }
+  }, []);
+
+  const getKeyboardOffset = useCallback(() => {
+    const scrollStep = gridCellSize * scale;
+    let dx = 0;
+    let dy = 0;
+
+    if (pressedKeys.current.has('ArrowUp')) dy += scrollStep;
+    if (pressedKeys.current.has('ArrowDown')) dy -= scrollStep;
+    if (pressedKeys.current.has('ArrowLeft')) dx += scrollStep;
+    if (pressedKeys.current.has('ArrowRight')) dx -= scrollStep;
+
+    return { dx, dy };
+  }, [scale]);
+
+  // Apply keyboard navigation offset if needed
+  const applyKeyboardNavigation = useCallback(() => {
+    const { dx, dy } = getKeyboardOffset();
+    if (dx !== 0 || dy !== 0) {
+      updateTranslation(dx, dy);
+    }
+  }, [getKeyboardOffset, updateTranslation]);
+
+  const handleKeyDown = useCallback(
+    (e: KeyboardEvent<SVGSVGElement>) => {
+      const isArrowKey = [
+        'ArrowUp',
+        'ArrowDown',
+        'ArrowLeft',
+        'ArrowRight',
+      ].includes(e.key);
+
+      if (isArrowKey) {
```

Comment:
Maybe could move this to some function, a bit repetitive as it happens the same for another method